src/gql/queries/get-patch.graphql

```diff
@@ -7,7 +7,7 @@ query Patch($id: String!) {
       baseVersionID
       githash
       id
-      projectID
```

Comment:
why do we need to change this field? will DownstreamTasks read this field?

Reply:
removed